Sources/AppAuth/macOS/OKTRedirectHTTPHandler.m

```diff
@@ -68,6 +68,12 @@ - (instancetype)initWithSuccessURL:(nullable NSURL *)successURL {
   return self;
 }
 
+- (void)setSuccessURL:(nullable NSURL *)successURL {
+    if (self) {
```

Comment:
Why the check for `if (self)` here? Self should be nonnull if the selector is capable of being invoked.